### ISSUE TITLE
Custom styling (rounded toasts, blocky toasts, custom background colours)

### ISSUE THREAD

**Issue by Script47** [Script47/Toast#4]
My plan was to have the main styles stored in the CSS so as to allow a quick override and also have a way to programmatically override all the styles of all the toasts. Something like:

```
$.toast.style('style-name', {
    border: '5px solid red'
});
```

The naming of the style would be optional and would be used in the instance that you had multiple styles and wanted to switch between them, for example:

```
$.toast({
    style: 'style-one'
});
```

or

```
$.toast({
    style: 'style-two'
});
```

Alternatively, you could pass the style directly to each toast:

```
$.toast({
    style: {
        border: '5px solid red'
    }
});
```

This is what I was thinking regarding this feature.

CC: @tip2tail

**Comment by joho1968** [comment]
Wouldn't it make sense to exhaust all Bootstrap native options (like rounded, etc) first, and then add "custom" stuff for cases not covered by Bootstrap?

**Comment by Script47** [comment]
@joho1968 agreed. I've been fairly busy recently but if you (or anyone else) could come with a new issue which contains a list of existing Bootstrap config options that are missing (and those that have been added in the plugin) it might be beneficial for myself and future pull requests.
